### ISSUE TITLE
The site is gone!

### ISSUE THREAD

**Issue by BrockA** [tms/stack-election#3]
Reference: https://stackapps.com/a/7519/7653

For https://elections.stackexchange.com/, Stack Exchange now gives:

> Couldn't find elections.stackexchange.com
>
> The Q&A site elections.stackexchange.com doesn't seem to exist…yet

The problem seems to be [since, at least, July 14th](https://stackapps.com/questions/2049/elections-statistics-page-for-stack-exchange-sites/7519#comment15936_2049).

**Comment by tms** [comment]
Yeah, it's been some time more than that unfortunately, the whole situation was a bit unexpected. I have preparations to get the site back up, but I haven't finished yet. This weekend I'll try to at least get _something_ back in place.

**Comment by BrockA** [comment]
Thanks!  Wish I could help, but my bandwidth/time is limited.

Also, reference:  https://meta.stackexchange.com/q/294719/148310

**Comment by PatMyron** [comment]
another SE election site came and went too:
https://github.com/Charcoal-SE/elections
